tests/workflow/interfaces/execute/test_jax.py

```diff
@@ -693,24 +693,26 @@ def test_max_diff(self, tol):
 
         def cost_fn(x):
             ops = [qml.RX(x[0], 0), qml.RY(x[1], 1), qml.CNOT((0, 1))]
-            tape1 = qml.tape.QuantumScript(ops, [qml.var(qml.PauliZ(0) @ qml.PauliX(1))])
+            tape1 = qml.tape.QuantumScript(
+                ops, [qml.var(qml.PauliZ(0) @ qml.PauliX(1))], shots=50000
+            )
 
             ops2 = [qml.RX(x[0], 0), qml.RY(x[0], 1), qml.CNOT((0, 1))]
-            tape2 = qml.tape.QuantumScript(ops2, [qml.probs(wires=1)])
+            tape2 = qml.tape.QuantumScript(ops2, [qml.probs(wires=1)], shots=50000)
 
             result = execute([tape1, tape2], dev, diff_method=param_shift, max_diff=1)
             return result[0] + result[1][0]
 
         res = cost_fn(params)
         x, y = params
         expected = 0.5 * (3 + jnp.cos(x) ** 2 * jnp.cos(2 * y))
-        assert np.allclose(res, expected, atol=tol, rtol=0)
+        assert np.allclose(res, expected, atol=2e-2, rtol=0)
```

Comment:
If the `tol` isn't being used, it can be removed as an argument to this test.

Reply:
seems to still work for line 719.

Reply:
Thanks for pointing that out :) Resolving.